jbmc/unit/java_bytecode/java_bytecode_convert_method/convert_method.cpp

```diff
@@ -518,3 +546,54 @@ SCENARIO(
     }
   }
 }
+
+SCENARIO(
+  "java convert method variable",
+  "[core][java_bytecode][java_bytecode_convert_method][variable]")
+{
+  symbol_tablet symbol_table;
+  java_string_library_preprocesst string_preprocess;
+  const class_hierarchyt class_hierarchy;
+  java_bytecode_convert_methodt converter{symbol_table,
+                                          null_message_handler,
+                                          10,
+                                          true,
+                                          {},
+                                          string_preprocess,
+                                          class_hierarchy,
+                                          false};
+
+  GIVEN("An int_array variable")
+  {
+    const source_locationt location;
+    const typet int_array_type = java_array_type('i');
+    const symbol_exprt int_array{"int_array", int_array_type};
+    const std::size_t variable_index = 0;
+    java_bytecode_convert_method_unit_testt::add_variable(
+      converter, variable_index, int_array, 0, 1, false, {});
```

Comment:
:pick: Since lots of similar typed parameters you might like to pull these out into named constants before:

```C++
const std::size_t start_pc =  0;
...
const bool is_parameter = false;
java_bytecode_convert_method_unit_testt::add_variable(
  converter, variable_index, int_array, start_pc, length, is_paramerter, {});
```

:point_left: